rdf4j/src/main/java/com/inrupt/client/rdf4j/RDF4JBodyHandlers.java

```diff
@@ -41,45 +43,91 @@
  */
 public final class RDF4JBodyHandlers {
 
+    private static void throwOnError(final Response.ResponseInfo responseInfo) {
+        if (!Response.isSuccess(responseInfo.statusCode())) {
+            throw new ClientHttpException(
+                    "Could not map to an RDF4J entity.",
```

Comment:
The real issue is that the request failed rather been an issue mapping it to a RDF4J entity. Is this message a bit misleading?